devito/dle/parallelizer.py

```diff
@@ -50,13 +50,21 @@ def functions(self):
         return (self.nthreads,)
 
 
-class SingleThreadProdder(Conditional, Prodder):
+class ThreadedProdder(Conditional, Prodder):
```

Comment:
Never quite sure what the difference between Prob and Prod is so gonna assume you chose the right one ^^